src/main/java/mate.academy/service/FileReaderService.java

```diff
@@ -1,7 +1,9 @@
 package mate.academy.service;
 
 import java.util.List;
+import mate.academy.lib.Component;
 
+@Component
```

Comment:
I don't think we need to put annotations over interfaces, can remove it.  I think only over implementations is enough(but up to you).

Reply:
tests are failed without annotaitions over the interfaces

Reply:
you should put this annotation above the class